kvmagent/kvmagent/plugins/tensorfusion/service.py

```
@@ -0,0 +1,212 @@
+        return True
+
+    def destroy_workers_by_vm(self, vm_uuid):
+        # type: (str) -> int
+        """Destroy all workers belonging to a VM. Returns count destroyed."""
+        workers = self._store.list_by_vm(vm_uuid)
+        count = 0
+        failures = []
+        for w in workers:
+            try:
+                destroyed = self.destroy_worker(w.device_uuid)
+            except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)